.travis.yml

```diff
@@ -14,6 +14,12 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - if [[ "`phpenv version-name`" != "7.1" ]]; then exit 0; fi
```

Comment:
This is how Travis manages the php version used… this includes the mayor + minor version, but not the release or any further appendices.